src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -0,0 +1,26 @@
+package de.retest.web;
+
+import java.awt.image.BufferedImage;
+
+import org.openqa.selenium.WebDriver;
+
+import com.assertthat.selenium_shutterbug.core.Shutterbug;
+import com.assertthat.selenium_shutterbug.utils.web.ScrollStrategy;
+
+public class ScreenshotProvider {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=101&issues=AWcNHOHWptIL6UVyV_nD&open=AWcNHOHWptIL6UVyV_nD)

[ISSUE_KEY]: AWcNHOHWptIL6UVyV_nD